pkg/estimator/server/server.go

```diff
@@ -291,7 +311,7 @@ func traceMaxAvailableReplicas(object string, start time.Time, request *pb.MaxAv
 			klog.Errorf("Failed to calculate cluster available replicas: %v", *err)
 			return
 		}
-		klog.Infof("Finish calculating cluster available replicas of resource(%s), max replicas: %d, time elapsed: %s", object, (*response).MaxReplicas, time.Since(start))
+		klog.V(2).Infof("Finish calculating cluster available replicas of resource(%s), max replicas: %d, time elapsed: %s", object, (*response).MaxReplicas, time.Since(start))
```

Comment:
it is better that the log level is greater than or equal to 4.

Reply:
This information may be important for estimator. This is the result of every estimation. If >= 4, logs will flood.

Reply:
but the log size is large when the cluster has too much node

Reply:
I'm afraid not. This log prints when estimator do an estimation once, not for all nodes.